WordPress/src/main/java/org/wordpress/android/sharedlogin/resolver/LocalMigrationOrchestrator.kt

```diff
@@ -47,9 +41,10 @@ class SharedLoginResolver @Inject constructor(
         }
         sharedLoginAnalyticsTracker.trackLoginStart()
         appPrefsWrapper.saveIsFirstTrySharedLoginJetpack(false)
-        val accessTokenCursor = getAccessTokenCursor()
-        if (accessTokenCursor != null) {
-            val accessToken = queryResult.getValue<String>(accessTokenCursor) ?: ""
+        val (accessToken) = localMigrationContentResolver.getDataForEntityType<AccessTokenData>(AccessToken)
+//        val accessTokenCursor = getAccessTokenCursor()
```

Comment:
Would it make sense to remove the commented block before merging? Or we want to handle it in another PR?

Reply:
Good question.. I left these commented lines since I didn't want to delete the existing error information. The error handling may be best to move to a separate PR, but I think that the existing work on errors will be reusable in the unified approach. So, this was like a "reminder" of a loose end that needs to be re-included somewhere. That said, I'm happy to remove these commented lines, and the existing names can be "recycled" / repurposed from the commit history :smile: .

Reply:
Handled with: https://github.com/wordpress-mobile/WordPress-Android/pull/17399/commits/1eb46108f518d578bdc4161d09de33766e11f14a